enterprise/server/certgenerator/certgenerator.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"context"
 	"crypto/ecdsa"
-	"crypto/rand"
+	crand "crypto/rand"
```

Comment:
I don't think this should be an aliased import -- we should not be importing math/rand in most of these files.

(If it did need to be aliased, the aliased imports are down below every other group of imports)